dungeon/assetlist.json

```diff
@@ -0,0 +1,26 @@
+{
+    "chain_id": "dungeon-1",
+    "assets": [
+      {
+        "description": "Dungeon native token",
+        "denom_units": [
+          {
+            "denom": "udgn",
+            "exponent": 0
+          },
+          {
+            "denom": "DGN",
+            "exponent": 6
+          }
+        ],
+        "base": "udgn",
+        "name": "Dungeon Coin",
+        "display": "DGN",
+        "symbol": "DGN",
+        "logo_URIs": {
+          "png": "./images/DGN.png"
```

Comment:
Please use a full url for the image reference

Reply:
updated